cypher/cypher-docs/src/docs/dev/ql/administration/constraints/index.asciidoc

```diff
@@ -93,8 +93,8 @@ If the constraint is not explicitly named, it will get an auto-generated name.
 The constraint name must be unique among both indexes and constraints.
 
 The command is optionally idempotent, with the default behavior to throw an error if you attempt to create the same constraint twice.
-With `IF NOT EXISTS`, no error is thrown and nothing happens should a constraint with the same name or same schema and constraint type already exist.
-It may still throw an error should a conflicting index or constraint exist.
```

Comment:
I think this sentence could maybe be expanded on a bit. I think it is not clear for a user what `conflicting` means, especially `conflicting constraint` given the sentence above. I guess that a conflicting constraint would be e.g. a node key constraint if a uniqueness constraint  on the same label and property already existed? We maybe don't have to list every case the can cause errors, but at least some examples.